packages/react/src/components/dropzone.tsx

```diff
@@ -1,6 +1,8 @@
 import { useCallback, useEffect, useState } from "react";
 import { twMerge } from "tailwind-merge";
 
+import type { FileWithPath } from "@uploadthing/dropzone/react";
```

Comment:
It feels a bit strange that this is where that type comes from, but probably fine.

Reply:
i don't see where else we'd get it from, unless we just make it a generic File instead?

Reply:
We could have an uploadthing/types package, but that makes it weird elsewhere, and also makes maintenance a pain. It's fine as is.

Reply:
I did standard File: https://github.com/pingdotgg/uploadthing/pull/524/commits/605fdb64819a74c51fdd07f513dbb4d96cfd7cdc